code/game/objects/effects/effect_system/effects_smoke.dm

```diff
@@ -69,6 +73,9 @@
 		return FALSE
 	if(C.smoke_delay)
 		return FALSE
+		addtimer(CALLBACK(src, PROC_REF(remove_smoke_delay), C), 10) //Sometimes during testing I'd somehow end up with a permanent smoke delay, so this is in case of that
```

Comment:
```suggestion
		addtimer(CALLBACK(src, PROC_REF(remove_smoke_delay), C), 10) //Sometimes during testing I'd somehow end up with a permanent smoke delay, so this is in case of that
```
goober i'm taking your weed

Reply:
I forgor